src/doc/rgw/rgw-qos.md

```diff
@@ -0,0 +1,32 @@
+# RGW Quality of Service
+## Requirements
+
+ - All RGWs should be aware how many requests an user or a bucket have been made per second
```

Comment:
we've explicitly considered that they may or may not -need- this, and more importantly, this feature is trying to make them -approximately aware-